x-pack/plugin/deprecation/src/test/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecksTests.java

```diff
@@ -590,4 +591,22 @@ public void testSharedDataPathSetting() {
                 "Found shared data path configured. Discontinue use of this setting."
             )));
     }
+
+    public void testClusterRoutingAllocationIncludeRelocationsSetting() {
+        Settings settings = Settings.builder()
+            .put(CLUSTER_ROUTING_ALLOCATION_INCLUDE_RELOCATIONS_SETTING.getKey(), false).build();
```

Comment:
I think both setting to false and true is deprecated, so suggest to randomize:

```suggestion
            .put(CLUSTER_ROUTING_ALLOCATION_INCLUDE_RELOCATIONS_SETTING.getKey(), randomBoolean()).build();
```